### PR TITLE
Outdoorindoor

### PR DESCRIPTION
**Pull request by prashant-glitch**
information about indoor wall salt lake wall (outdoor wall) and natural practice have been added.
The entire component (OutdoorIndoor) is added in-home page.

### CODE REVIEW

**Comment by prashant-glitch** [comment]
page view has been changed for indooroutdoor content, that will be available in home page

**Comment by satyam0601** [comment]
put outdoorindoor in home folder and refactor code

**Comment by prashant-glitch** [comment]
css and structure of outdoorindoor component has been changed

File: src/components/Home/OutdoorIndoor/OutdoorIndoor.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import './OutdoorIndoor.css';
+import ContentContainer from '../../Common/ContentContainer/ContentContainer';
+function OutdoorIndoor()
+{
+  return(
+    <>
+    <ContentContainer>
```
Comment (satyam0601) [inline src/components/Home/OutdoorIndoor/OutdoorIndoor.jsx]: remove this content container as home page is already enclosed in it

Comment (prashant-glitch) [reply src/components/Home/OutdoorIndoor/OutdoorIndoor.jsx]: Done

**Comment by prashant-glitch** [comment]
ContentCotainer is removed from outdoorindoor

**Pull request merged**